Gems/Multiplayer/Code/Tests/ServerHierarchyTests.cpp

```diff
@@ -433,8 +433,9 @@ namespace Multiplayer
     TEST_F(ServerDeepHierarchyTests, Testing_Limiting_Hierarchy_Maximum_Size)
     {
         uint32_t currentMaxLimit = 0;
-        m_console->GetCvarValue<uint32_t>("bg_hierarchyEntityMaxLimit", currentMaxLimit);
-        m_console->PerformCommand("bg_hierarchyEntityMaxLimit 2");
+        auto console = AZ::Interface<AZ::IConsole>::Get();
```

Comment:
You can move this to the base test, just as m_console was declared. just save the pointer there the same way, when it is ready.

Reply:
It's used in two tests, I didn't see much benefit to storing it off.